### PR TITLE
Simplifying profit sharing

### PR DESCRIPTION
**Pull request by dmurko**
Ref https://github.com/niteoweb/operations/issues/632

- Title capitalization.
- Removing the table of contents.
- Loans are Business, not Personal.
- Donations to organizations are and should be listed on https://niteo.co/about

### CODE REVIEW

File: people/profit-sharing.md [hunk 1]
```diff
@@ -22,8 +14,7 @@ The profit will only be paid-out if cash reserves are high enough to cover 12 mo
 This part rewards the people that have believed in the company in the past, have invested in it, have lent money to it, etc. The priority of payouts are as follows:
 
 1. Bank loans (to decrease interest rates)
-2. Personal loans
-3. Investor payouts
+2. Business loans
 4. Founder payouts
```
Comment (zupo) [inline people/profit-sharing.md]: should be `3.` now?

File: people/profit-sharing.md [hunk 2]
```diff
@@ -53,11 +44,3 @@ The remaining part of the profit share is what we donate to charities and causes
...
 
 The organization with the most total "points" (current quarter plus previous quarters) gets the profit share donation. Then its points are reset to zero (if the organization wins by more than 1 point, the difference is carried over to next month). Other organizations keep their points for the next quarter. If there is a tie, the Partners decide the winner.
-
-So far, we have donated to the following organizations:
-
-* [Let's Encrypt](https://letsencrypt.org/)
-* [Botrstvo [SI]](https://www.boter.si/)
-* [LibreOffice](https://www.libreoffice.org/)
-* [PodCrto [SI]](http://podcrto.si/)
-* [Wikipedia](https://www.wikipedia.org/)
```
Comment (zupo) [inline people/profit-sharing.md]: Link to niteo.co where there are list of donations.

File: people/profit-sharing.md [hunk 3]
```diff
@@ -1,16 +1,8 @@
...
 
 This document describes how we manage the distribution of the company's profit, on a quarterly basis.
 
-  **Contents**
-  1. [The concept](#the-concept)
-  2. [Past](#past)
-  3. [Present](#present)
-  4. [Future](#future)
-  5. [Donations](#donations)
-
```
Comment (zupo) [inline people/profit-sharing.md]: I kinda like the ToCs at the top ...

Comment (dmurko) [reply people/profit-sharing.md]: Then we need to have them everywhere and it adds an additional complexity to the doc. We can have them but I would avoid it. IMO the doc should be short or clear enough that it doesn't need them.

**Comment by dmurko** [comment]
@zupo Review again please.

**Pull request merged**